packages/components/src/components/contract-card/contract-card-items/accumulator-card-body.tsx

```diff
@@ -1,13 +1,46 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { isCryptocurrency, getLimitOrderAmount, isValidToSell } from '@deriv/shared';
 import ContractCardItem from './contract-card-item.jsx';
 import ToggleCardDialog from './toggle-card-dialog.jsx';
-import Icon from '../../icon';
-import MobileWrapper from '../../mobile-wrapper';
-import Money from '../../money';
+import Icon from '../../icon/index';
+import MobileWrapper from '../../mobile-wrapper/index';
+import Money from '../../money/index';
 import { ResultStatusIcon } from '../result-overlay/result-overlay.jsx';
+import { ProposalOpenContract, Proposal, ContractUpdate } from '@deriv/api-types';
+import { TIsValidToSell } from '@deriv/shared/src/utils/contract/contract-types';
+
+type TToastConfig = {
+    key?: number;
+    content: string;
+    timeout?: number;
+    is_bottom?: boolean;
+    type?: string;
+};
+
+export type TContractInfo = Pick<ProposalOpenContract, 'buy_price' | 'sell_price' | 'contract_id'> &
+    Required<Pick<ProposalOpenContract, 'profit'>> & {
+        limit_order?: Proposal['limit_order'];
+    } & TIsValidToSell;
+
+type TAccumulatorCardBody = {
+    addToast?: (toast_config: TToastConfig) => void;
+    connectWithContractUpdate?: (Component: JSX.Element) => unknown;
+    contract_info: TContractInfo;
+    contract_update: ContractUpdate;
+    currency: Required<ProposalOpenContract>['currency'];
```

Comment:
do we really need to mark it as required? without optional ? Would it be required? and same on line 22